src/api/bannerAdmin.ts

```diff
@@ -1,16 +1,15 @@
 import firebase from "firebase/app";
+import { BannerFormData } from "types/banner";
 
-export const makeUpdateBanner = (
-  venueId: string,
-  onError: (errorMsg: string) => void
-) => async (message?: string): Promise<void> => {
+export const makeUpdateBanner = (venueId: string) => async (
+  data: BannerFormData
+): Promise<void> => {
   const params = {
     venueId,
-    bannerMessage: message ?? "",
+    banner: data,
   };
```

Comment:
Is there a reason to have this function as a factory? () => () => void. Should we just make it () => void.

data prop could be renamed into `banner`

Reply:
https://github.com/sparkletown/sparkle/pull/1440/commits/d64cc297dd104ef219a5c7970087b8fc1b3c7b1c